src/components/Form/MucSelectItem.vue

```diff
@@ -0,0 +1,25 @@
+<template>
+  <span>
+    {{ itemText }}
+  </span>
+</template>
+
+<script setup lang="ts">
+import { computed } from "vue";
+
+import { ItemAsObject } from "./MucSelectTypes";
+
+const props = defineProps<{
+  item: string | ItemAsObject;
```

Comment:
```suggestion
  item: MucSelectItemTypes;
```
I think you forgot one here + import :)